packages/db/src/schema/environment.ts

```diff
@@ -90,11 +93,16 @@ export const environmentPolicy = pgTable("environment_policy", {
   concurrencyType: concurrencyType("concurrency_type").notNull().default("all"),
   concurrencyLimit: integer("concurrency_limit").notNull().default(1),
 
-  duration: bigint("duration", { mode: "number" }).notNull().default(0),
+  // Duration in milliseconds over which to gradually roll out releases to this
```

Comment:
wait are we doing seconds or milliseconds

Reply:
we can do seconds after we migrate this, low p